### DISCUSSION TITLE
Android→Adronid12，Black screen when unlocking，Using an operating system password doesn't help either

### DISCUSSION THREAD

**Discussion by Mgrsc** [rustdesk/rustdesk#8602] [General]
### Bug Description

Android→Adronid12，Black screen when unlocking，Using an operating system password doesn't help either

### How to Reproduce

Appears when connected to my phone

### Expected Behavior

Enter the lock screen password manually or enter the lock screen password automatically to enter the system

### Operating system(s) on local side and remote side

Android12→Android12

### RustDesk Version(s) on local side and remote side

1.2.6→1.2.6

### Screenshots

![Screenshot_20240704_021207_com carriez flutter_hbb](https://github.com/rustdesk/rustdesk/assets/118801216/c770b750-46f0-482b-8acd-933ce8f1a70e)
![Screenshot_20240704_021200_com carriez flutter_hbb](https://github.com/rustdesk/rustdesk/assets/118801216/3cb67ff6-685d-4274-aa00-d49756479acc)

**Comment by rustdesk** [comment]
we have no ability to handle android lock screen yet.

**Comment by cxwithyxy** [comment]
有另一个方式绕开这个问题：你可以用另一个手机给这个手机打电话，响铃时屏幕会亮起，此时rustdesk就可以远程连接了
